aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,29 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+
+## January 24 - Febuary 4, 2022
```

Comment:
Thinking we should extend this to cover last week's activities as well. If we are publishing, we should publish the most timely information.

Reply:
I think we have all notable items covered from last week in the suggestions made below.

```suggestion
## January 24 - February 11, 2022
```

Reply:
sounds good